t/10_susedistribution.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+# Copyright @ SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use testapi;
+use Test::MockModule;
+use Test::MockObject;
```

Comment:
This module isn't used, so I would remove it:
```suggestion
```

Reply:
*ready for review*